apps/artwork/components/commercial/templates/price.jade

```diff
@@ -1,10 +1,11 @@
+- var showPriceLabel = artwork.price && artwork.availability !== 'sold'
```

Comment:
Basically we just render `sale_message` directly now, except in the case of the `sale_message` containing some sort of pricing information (and not for sold works). In that case, we want to still just render the `sale_message`, but format it differently w/ some pricing labels/tooltips.

<img width="325" alt="screen shot 2017-02-01 at 2 32 13 pm" src="https://cloud.githubusercontent.com/assets/1457859/22522873/3fd6f236-e88b-11e6-820c-30ce6a6c601e.png">